.github/workflows/stub_23.lts.1+.yaml

```diff
@@ -0,0 +1,23 @@
+name: stub_23.lts.1+
```

Comment:
Why do we want to name this workflow (and the other equivalent ones) `stub_23.lts.1+` instead of just `stub` (as it's on the 23.lts.1+ branch already)?

Reply:
if we name it 'stub.yaml' it will show up as 'stub' under workflows, users will see two stub workflows (main and 23.lts.1+) - this would be super confusing for users.

Reply:
Wasn't thinking about how they all showed up next to each other on that page—that makes sense then

Reply:
You know what? I think you are on to something, workflow names come from 'name:' field in the yaml file itself, not the filename.

Reply:
ah no, can't have same workflow file names across branch - they collide and replace each other in GitHub UI every time each runs :(